src/skin/options-layout.css

```diff
@@ -153,6 +153,16 @@ header, #tabs {
     margin-left: 25px;
 }
 
+#hide-widgets-row {
+    width: 100%;
+    margin-left: 25px;
+    margin-top: 5px;
+}
+
+#hide-widgets-select {
+  width: 500px;
```

Comment:
I think this is too large for mobile (Firefox for Android)/small displays. Previously: #2195.

Reply:
This is what it looks like after updating the width to 300 px:

![image](https://user-images.githubusercontent.com/8368015/66852355-63a2bb80-ef42-11e9-9af8-a0d4a739d06f.png)